README.md

```diff
@@ -3691,6 +3692,78 @@ FLAG DESCRIPTIONS
     to persistently set a default project for all commands that accept this flag.
 ```
 
+## `mw mail address update MAILADDRESS-ID`
+
+Update a mail address
+
+```
+USAGE
+  $ mw mail address update MAILADDRESS-ID -a <value> [-q] [--catch-all] [--quota <value>] [--password <value>]
+    [--random-password] [--forward-to <value>]
+
+ARGUMENTS
+  MAILADDRESS-ID  ID or mail address of a mailaddress
+
+FLAGS
+  -a, --address=<value>        (required) mail address
+  -q, --quiet                  suppress process output and only display a machine-readable summary.
+      --catch-all              make this a catch-all mail address
+      --forward-to=<value>...  forward mail to another address
```

Comment:
Documentation question: Are the `--forward-to` addresses additive, or do these values replace already existing forward addresses?

Reply:
They completely replace the existing array of forward-addresses.